apps/timetable/admin/index.html

```diff
@@ -0,0 +1,150 @@
+
+        <script id="gh-subheader-pickers-template" type="text/template">
+            <a class="pull-right gh-show-video" href="#" style="display: none;"><i class="fa fa-youtube-play"></i> Help video<a>
+            <%= _.partial('subheader-pickers') %>
+        </script>
+
+        <script id="gh-main-tripos-template" type="text/template">
+            <div id="gh-main-tripos">
+                <div class="gh-video">
+                    <button class="btn gh-hide-video"><i class="fa fa-remove"></i></button>
+                    <span class="gh-video-info">A quick look at how to use<br/>the new timetable system</span>
+                    <button class="btn gh-play-video"><i class="fa fa-youtube-play"></i>Play video</button>
+                </div>
+                <div class="row gh-row-part">
+                    <div class="col-md-4 gh-part">
+                        <p class="gh-part-editing">Mary Wellingtion is editing</p>
+                        <p class="gh-part-tripos">Asian and Middle Eastern Studies - Japanese Studies</p>
+                        <p class="gh-part-part">Part IA</p>
+                        <button class="btn btn-default pull-left gh-part-edit">Edit</button>
+                        <span class="gh-part-draft pull-right"><i class="fa fa-eye-slash"></i> Draft</span>
+                    </div>
+                    <div class="col-md-4 gh-part">
+                        <p class="gh-part-tripos">Asian and Middle Eastern Studies - Japanese Studies</p>
+                        <p class="gh-part-part">Part IB</p>
+                        <button class="btn btn-default pull-left gh-part-edit">Edit</button>
+                    </div>
+                    <div class="col-md-4 gh-part">
+                        <p class="gh-part-tripos">Asian and Middle Eastern Studies - Japanese Studies</p>
+                        <p class="gh-part-part">Part II</p>
+                        <button class="btn btn-default pull-left gh-part-edit">Edit</button>
+                    </div>
+                </div>
+                <div class="row gh-row-part">
+                    <div class="col-md-4 gh-part">
+                        <p class="gh-part-tripos">Asian and Middle Eastern Studies - Chinese Studies</p>
+                        <p class="gh-part-part">Part IA</p>
+                        <button class="btn btn-default pull-left gh-part-edit">Edit</button>
+                    </div>
+                    <div class="col-md-4 gh-part">
+                        <p class="gh-part-tripos">Asian and Middle Eastern Studies - Chinese Studies</p>
+                        <p class="gh-part-part">Part IB</p>
+                        <button class="btn btn-default pull-left gh-part-edit">Edit</button>
+                    </div>
+                </div>
+            </div>
+        </script>
+
+        <script data-main="/shared/gh/api/gh.bootstrap.js" data-loadmodule="/admin/ui/js/index.js" src="/shared/vendor/js/require-jquery.js"></script>
```

Comment:
If we point it to `/apps/timetable/admin/js/index.js` could we eliminate the `admin` entry in the apache config? It'd also make it clearer that we're not pointing to the admin app files.